contents/tutorials/nextjs-cookie-banner.md

```diff
@@ -55,13 +66,13 @@ export default function RootLayout({ children }) {
 }
 ```
 
-After setting this up and running `npm run dev`, PostHog starts autocapturing events, but a PostHog-related cookie is set for the user without their consent. So, let's set up a cookie consent banner!
+After setting this up and running `npm run dev`, PostHog starts autocapturing events, but a PostHog-related cookie is set for the user without their consent.
 
 ![Cookie set](../images/tutorials/nextjs-cookie-banner/cookie.png)
 
 ## Creating a cookie banner
 
-Create another file in the `app` folder named `banner.js`for the banner component with a bit of text explaining cookies and buttons to accept or decline.
+Create another file in the `app` folder named `banner.js` for the banner component with a bit of text explaining cookies and buttons to accept or decline.
 
 Importantly, to avoid a hydration error, we must check if the frontend has mounted and only show the component if so. We can use `useState` and `useEffect` to do this. Together, this looks like this:
```

Comment:
we can remove this if the initial state is true and not false?

Reply:
If we set the initial state to true, it flickers the cookie banner when hidden, showing up briefly before going away.